index.html

```diff
@@ -16,8 +16,14 @@
     communities are private, so you'll need to request an invite to join.  The individual communities
     are responsible for managing the invitation and vetting process.
   </p>
-  <ul>{% for community in site.data.communities %}
-    <li><a href="{{ community.url }}">{{ community.name }}</a></li>{% endfor %}
+  <ul>
+    {% for community in site.data.communities | sort 'name' %}
```

Comment:
Does this sort syntax work?  I'm getting a syntax error when running this locally.